### ISSUE TITLE
model.compile("L-BFGS")     Incorrect losses and termination of training

### ISSUE THREAD

**Issue by ShupengGao** [lululxvi/deepxde#1208]
Hello,Lu
After training with the example given, the following error occurred. I don't know why this happened.
This is not an example, multiple examples have the same error.

'compile' took 0.000324 s

Training model...

Step      Train loss                        Test loss                         Test metric
15000     [2.39e-03, 1.75e-05, 1.60e-03]    [2.39e-03, 1.75e-05, 1.60e-03]    []
16000     [1.59e-03, 4.87e-06, 1.20e-03]    [1.59e-03, 4.87e-06, 1.20e-03]    []  
17000     [0.00e+00, 2.77e+02, 1.57e+02]    [0.00e+00, 2.77e+02, 1.57e+02]    []  
18000     [0.00e+00, 1.70e+12, 3.68e+12]    [0.00e+00, 1.70e+12, 3.68e+12]    []  
19000     [0.00e+00, 1.21e+36, 3.47e+35]    [0.00e+00, 1.21e+36, 3.47e+35]    []  
20000     [nan, nan, nan]                   [nan, nan, nan]                   []

**Comment by ShupengGao** [comment]
The issue is solved by using float64 #28 .

**Comment by kmache** [comment]
I have the same problem. I am using float64 but the same issue still happen
